crates/iroha_data_model/src/block.rs

```diff
@@ -52,6 +52,9 @@ mod model {
         /// Number of blocks in the chain including this block.
         #[getset(get_copy = "pub")]
         pub height: NonZeroU64,
+        /// Number of non-empty blocks in the chain including this block.
+        #[getset(get_copy = "pub")]
+        pub height_non_empty: NonZeroU64,
```

Comment:
I understand that `height_non_empty` could be useful for inspecting blocks, but I'm not sure if it's worth adding to the block headers

Reply:
Can this be merged if this field is removed?

Reply:
Removed `height_non_empty` from `BlockHeader`